packages/contract-tests/test/errors.js

```diff
@@ -3,7 +3,7 @@ var util = require("./util");
 
 describe("Client appends errors (vmErrorsOnRPCResponse)", function () {
   var Example;
-  var providerOptions = { vmErrorsOnRPCResponse: true }; // <--- TRUE
+  var providerOptions = { legacyInstamine: true, vmErrorsOnRPCResponse: true }; // <--- TRUE
```

Comment:
`legacyInstamine` mode must be enabled whenever `vmErrorsOnRPCResponse` is `true`